docs/events/openfoodfacts-query.yaml

```diff
@@ -0,0 +1,18 @@
+asyncapi: "3.0.0"
+
+info:
+  title: Events that off-query subscribes to
+  version: 0.0.1
+
+servers:
+  redis:
+    title: Redis server
+    host: redis:6379
+    protocol: redis
+
+operations:
+  product_updated:
+    action: receive
+    channel:
+      messages:
+        $ref: "https://raw.githubusercontent.com/openfoodfacts/openfoodfacts-server/refs/heads/redis-docs/docs/events/messages/product_updated.yaml"
```

Comment:
Blocker to update the link to the final commit after openfoodfacts/openfoodfacts-server#12258 has been merged